MSAL/src/public/MSALAccount.h

```diff
@@ -28,6 +28,15 @@
 #import <Foundation/Foundation.h>
 
 @class MSALAccountId;
+@class MSALTenantProfile;
+
+typedef NS_ENUM(NSInteger, MSALAccountType)
+{
+    MSALAccountTypeAAD,
+    MSALAccountTypeB2C,
+    MSALAccountTypeMSA,
+    MSALAccountTypeUnknown
+};
 
 @interface MSALAccount : NSObject <NSCopying>
```

Comment:
Should `MSALAccount` be the abstract class (deny it's initialization)?

Reply:
deleted all the subclasses seems we don't need them :)